docs/guide/search/extend_solr.md

```diff
@@ -70,74 +70,13 @@ which are children of the webinar.
 The field mapper could then look like this:
 
 ```php
- <?php
-
-namespace App\Search\Mapper;
-
-use EzSystems\EzPlatformSolrSearchEngine\FieldMapper\ContentFieldMapper;
-use eZ\Publish\SPI\Persistence\Content\Handler as ContentHandler;
-use eZ\Publish\SPI\Persistence\Content\Location\Handler as LocationHandler;
-use eZ\Publish\SPI\Persistence\Content;
-use eZ\Publish\SPI\Search;
-
-class WebinarEventTitleFulltextFieldMapper extends ContentFieldMapper
-{
-    /**
-     * @var \eZ\Publish\SPI\Persistence\Content\Type\Handler
-     */
-    protected $contentHandler;
-
-    /**
-     * @var \eZ\Publish\SPI\Persistence\Content\Location\Handler
-     */
-    protected $locationHandler;
-
-    /**
-     * @param \eZ\Publish\SPI\Persistence\Content\Handler $contentHandler
-     * @param \eZ\Publish\SPI\Persistence\Content\Location\Handler $locationHandler
-     */
-    public function __construct(
-        ContentHandler $contentHandler,
-        LocationHandler $locationHandler
-    ) {
-        $this->contentHandler = $contentHandler;
-        $this->locationHandler = $locationHandler;
-    }
-
-    public function accept(Content $content)
-    {
-        // ContentType with ID 42 is webinar event
-        return $content->versionInfo->contentInfo->contentTypeId == 42;
-    }
-
-    public function mapFields(Content $content)
-    {
-        $mainLocationId = $content->versionInfo->contentInfo->mainLocationId;
-        $location = $this->locationHandler->load($mainLocationId);
-        $parentLocation = $this->locationHandler->load($location->parentId);
-        $parentContentInfo = $this->contentHandler->loadContentInfo($parentLocation->contentId);
-
-        return [
-            new Search\Field(
-                'fulltext',
-                $parentContentInfo->name,
-                new Search\FieldType\FullTextField()
-            ),
-        ];
-    }
-}
+[[= include_file('code_samples/front/search_extensibility/solr/src/FieldMapper/WebinarEventTitleFulltextFieldMapper.php') =]]
```

Comment:
`code_samples/front/` was supposed to cover rendering on the front page in general, here maybe `code_samples/search/solr/...`?